src/schemas/data/attributes.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "object",
+  "description": "Attributes",
+  "title": "IAttributes",
+  "required": ["_type", "label"],
+  "additionalProperties": false,
+  "properties": {
+    "_type": {
```

Comment:
Do we need `_type`?

Reply:
Since there are several types of attributes, I would say we should keep them.